model/import/Validator/HeaderValidator.php

```diff
@@ -149,4 +149,9 @@ private function findOccurrences(array $header, string $headerRegex): array
 
         return $occurrences;
     }
+
+    protected function getErrorMessagePrefix(): string
+    {
+        return __('Header');
```

Comment:
This prefix is only required for the header and can be added directly here. Lets remove this protected method from all implementation, please :)